Source/WebCore/Modules/WebGPU/GPUBindGroup.cpp

```diff
@@ -28,6 +28,11 @@
 
 namespace WebCore {
 
+GPUBindGroup::~GPUBindGroup()
+{
+    m_backing->destroy();
```

Comment:
I think, either:
A) the mapping between WebCore types and WebKit types is one-to-one, in which case the destructor for the webkit type will be called here without us having to do anything in WebCore, or
B) the mapping between them isn't one-to-one, and something else has retained a WebKit type, and that thing will be left with a dangling pointer after this code executes.

So I kind of feel that WebCore should be left alone here, and destruction should be driven by WebKit objects's destructors in the web process.